eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -56,8 +59,9 @@ jobs:
           /p:ServiceDirectory=${{ parameters.ServiceToTest }}
           /p:IncludeSrc=false /p:IncludeSamples=false /p:IncludePerf=false /p:IncludeStress=false
           /p:RunApiCompat=false /p:InheritDocEnabled=false
-          /p:Configuration=$(BuildConfiguration) $(ConvertToProjectReferenceOption)
```

Comment:
We should probable remove https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/variables/globals.yml#L4 as well as I don't see any other usages of this variable anymore.